### ISSUE TITLE
Error when trying to open KEYPM "Can not open the file "$STRT_REPLACE$""

### ISSUE THREAD

**Issue by seanvree** [webprofusion/certify#322]
OS:  Windows 10E x64
Certify ver:  Community edition 3.0.11.30336

ISSUE:

I was able to renew my certs and bind to IIS, however, when I browse to the cert folder  C:\ProgramData\ACMESharp\sysVault\45-KEYPM and try to open the key.pem file, I get the error:  

```
Can not open the file "$STRT_REPLACE$"
```

all other files are fine. 

![image](https://user-images.githubusercontent.com/8906637/41199082-858c3306-6c40-11e8-8873-306beaf76fa3.png)

**Comment by seanvree** [comment]
Yeah, actually, I was opening as admin, even then, didn't work.  However...did this:

https://github.com/webprofusion/certify/issues/269

Come to find out it was my issue.  DERP. 


But of note, even as local admin, doesn't' work, you have to use psexec, then run:

```
CIPHER /d /a /I /s:”[key.file.name]”
```

**Comment by webprofusion-chrisc** [comment]
Thanks, forgot about that, yes Local System owns those. Security through obscurity :) - this reminds me that the upcoming v4 doesn't support EFS yet. Out of interest, what workflow are you trying to achieve (with the PEM etc)?

**Comment by seanvree** [comment]
@webprofusion-chrisc  I just needed to copy the .pem file to use with a couple other services on the box, not just IIS.  But yeah, might be a good wiki addition. 

Thanks brotha.
